scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -78,9 +81,35 @@ def open_spider(self, spider: Spider) -> None:
         if hasattr(spider, "download_warnsize"):
             self._warn_size = spider.download_warnsize
 
+    @staticmethod
+    def _get_unsupported_suggestion(encoding: bytes) -> str:
+        if encoding == b"br":
+            return " You need to install brotli or brotlicffi to use it."
+        if encoding == b"zstd":
+            return " You need to install zstandard to use it."
+        return ""
```

Comment:
Seems unused.

Reply:
Indeed.

TBH the code I pushed is "sample" one, as different design decisions need quite different code. That's also why there are no tests yet.